src/main/java/com/tumblr/jumblr/JumblrClient.java

```diff
@@ -342,7 +343,8 @@ public Post postReblog(String blogName, Long postId, String reblogKey, Map<Strin
         Map<String, Object> soptions = JumblrClient.safeOptionMap(options);
```

Comment:
@KevinTCoughlin can you update the javadoc for `blogName` while you're at it? It should be "the name of the blog to post _to_"